### PR TITLE
[DOCFIX] Clarify steps to configure Alluxio on k8s

### PR DESCRIPTION
**Pull request by ZhuTopher**
I added a section to point users towards the configuration steps for Alluxio workers. Without configuring/disabling Alluxio worker short-circuit access, the Worker Pods will never start.

### CODE REVIEW

**Comment by LuQQiu** [comment]
@jiacheliu3 PTAL, thanks!

**Review by madanadit — APPROVED** [review]
Thanks for this improvement!

File: docs/en/deploy/Running-Alluxio-On-Kubernetes.md [hunk 1]
```diff
@@ -81,9 +81,15 @@ spec:
...
     path: /tmp/alluxio-journal-0
 ```
->Note: By default each journal volume should be at least 1Gi, because each Alluxio master Pod
+Note:
+- By default each journal volume should be at least 1Gi, because each Alluxio master Pod
 will have one PersistentVolumeClaim that requests for 1Gi storage. You will see how to configure
 the journal size in later sections.
+- If this `hostPath` is not already present on the host, Kubernetes will create it. However
+the assigned user:group permissions may prevent the Alluxio masters & workers from accessing it.
+Please ensure the permissions are set to allow the pods to access the directory.
```
Comment (jiacheliu3) [inline docs/en/deploy/Running-Alluxio-On-Kubernetes.md:90]: nit: you might wanna add a link to https://kubernetes.io/docs/concepts/storage/volumes/#hostpath

**Review by jiacheliu3 — APPROVED** [review]
LGTM with a nit comment, thanks for the improvement!

**Comment by madanadit** [comment]
alluxio-bot, merge this please.

**Comment by madanadit** [comment]
alluxio-bot, cherry-pick this to branch-2.5 please.

**Comment by LuQQiu** [comment]
alluxio-bot, cherry-pick this to branch-2.5 please

**Pull request merged**